yt_dlp/dependencies/__init__.py

```diff
@@ -43,6 +43,11 @@
 
 try:
     import sqlite3
+    # `sqlite3.version` is deprecated in py3.12, see https://github.com/yt-dlp/yt-dlp/issues/8152
+    import warnings as _warnings
+    with _warnings.catch_warnings():
+        _warnings.simplefilter('ignore', DeprecationWarning)
+        sqlite3._yt_dlp__version = getattr(sqlite3, 'version', '2.6.0+')
```

Comment:
This version has no meaning, hence why it is being deprecated. I would set it to None and fix the optional dep debug statement to not add the version suffix in that case.